tests/acceptance/15_control/02_agent/agentaccess.cf.sub

```diff
@@ -0,0 +1,10 @@
+body agent control
+{
+    agentaccess => { "nobody" };
+}
+
+bundle agent main
+{
+  reports:
+    "I am here!";
```

Comment:
Add a comment to explain why this is printed, and what is expected behavior in this test.